S/SuiteSparse/ParU/build_tarballs.jl

```diff
@@ -33,4 +33,4 @@ products = [
 
 
 build_tarballs(ARGS, name, version, sources, script, platforms, 
-               products, dependencies; julia_compat="1.11",preferred_gcc_version=v"9")
+               products, dependencies; julia_compat="1.12",preferred_gcc_version=v"9")
```

Comment:
You can't change compat bound without changing version number.  You aren't changing anything here, why doing this?

Reply:
Discussed this offline: https://github.com/JuliaRegistries/General/pull/118860 wasn't actually merged, so changing this compat bound is safe.